app/api/datastore/v2/reviewing.rb

```diff
@@ -23,6 +22,21 @@ class Reviewing < Base
             end
           end
         end
+
+        desc 'Mark an application as complete.'
+        params do
+          requires :application_id, type: String, desc: 'Crime Application UUID'
+        end
+
+        route_param :application_id do
+          resource :complete do
+            put do
+              complete_params = declared(params).symbolize_keys
```

Comment:
If the only param in this endpoint is `application_id` we probably don't need this, and we can replace it with just `params[:application_id]`
Unless you are expecting some additional params will be added in another PR, then it makes sense 👍